elixir/apps/web/lib/web/live/resources/show.ex

```diff
@@ -340,4 +334,14 @@ defmodule Web.Resources.Show do
       {:noreply, push_navigate(socket, to: ~p"/#{socket.assigns.account}/resources")}
     end
   end
+
+  defp http_link?(address_description) do
+    uri = URI.parse(address_description)
```

Comment:
Would this throw an error if `address_description` is `nil`?

Reply:
Yeah, updated